src/main/scala/com/gu/fastly/Lambda.scala

```diff
@@ -92,6 +96,9 @@ class Lambda {
     println(s"Sent $purgeType purge request for content with ID [$contentId], service with ID [$serviceId] and surrogate key [$surrogateKey]. Response from Fastly API: [${response.code}] [${response.body.string}]")
 
     val purged = response.code == 200
+
+    sendPurgeCountMetric
```

Comment:
While we're at it, is it worth counting based on `response.code`? E.g. success vs fail. Or perhaps that's something to consider once we've determined that this technique works. Either way I'm going to approve this :)

Reply:
Yeah - this PR it literally about getting CloudWatch to work...
Then extend the metric with Dimensions.